contents/python/numpy.md

```diff
@@ -0,0 +1,167 @@
+<frontmatter>
+  title: NumPy
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
+# NumPy
+
+Author: [Jeremy Tan](https://github.com/Parcly-Taxel)
+
+<box id="article-toc">
+* [What NumPy can and cannot do for you](#what-numpy-can-and-cannot-do-for-you)
+* [The array type](#the-array-type)
+* [Library functions‎](#library-functions)
+* [Getting started‎](#getting-started)
+* [Resources‎](#resources)
+</box>
+
+Data scientists often work with highly structured data, like time series or plots of pixel values. On these data sets,
+which can reach into the gigabyte range and beyond, they routinely need to perform operations uniformly and efficiently.
+
+Through list comprehension, Python somewhat satisfies these requirements, but not satisfactorily.
```

Comment:
See if you can state this point without mentioning list comprehensions, to reduce jargon. e.g., 
>Python built-in features are already somewhat optimized for scientific computing but may not be good enough for heavy ...